src/internal/clusterstate/v2.4.0.go

```diff
@@ -17,13 +20,25 @@ var state_2_4_0 migrations.State = state_2_3_0.
 	Apply("Add default project", func(ctx context.Context, env migrations.Env) error {
 		var defaultProject = &pfs.ProjectInfo{
 			Project: &pfs.Project{
-				Name: pfs.DefaultProjectName,
+				Name: "", // hardcoded so that pfs.DefaultProjectName may change in the future
```

Comment:
Because CORE-989 was completed beforehand, and this migration applies after that. You raise a good issue, though: since these changes are being made between rather than across releases, perhaps the default project can just be created with the name `"default"` here; anyone running a dev or alpha version can just rebuild his database, right?

Reply:
Yea, I would prefer to just use `"default"` here.

Reply:
I’ve updated to use `"default"`, and consequently removed the migration of the default project itself.